client/webserver/site/src/js/markets.js

```diff
@@ -815,8 +815,8 @@ export default class MarketsPage extends BasePage {
         Doc.hide(status)
         status.textContent = 'canceled'
       } else if (note.subject === 'revoke') {
+        Doc.hide(status)
```

Comment:
I'm still struggling to get the order to say "revoked" in the Your Orders table.

Reply:
Despite [adding an `newOrderNote("revoke",...)` to `(*trackedTrade).revoke`](https://github.com/decred/dcrdex/pull/705/files#diff-ca928c370ec8b724badec3db0dbd7bc4R990-R999), and tweaking this js, I've failed to get the order to go from "booked" to "revoked" even though with the frontend logger I see the order note and the book gets cleared (chart and buy/sell tables).  @buck54321 I'm afraid I might need a pointer from you to figure out why the frontend isn't acting on the order note.  It does seem to be matching the target ID in metaOrders, so I'm at a loss.

Reply:
It looks like you're hiding the status right before you set the `textContent`, which is exactly what is happening a couple of lines up, but I'm not sure why. But note that you are trying to update the `cancelStatus`, not the order status column. What you really need is a note containing info for the **trade**, where the `revoke` subject note contains info for the **cancel**.